src/Logging/PsrLogger.php

```diff
@@ -241,8 +241,10 @@ public function log($level, $message, array $context = [])
 
         $entry = $this->logger->entry(
             $message,
-            $this->resource,
-            $context + ['severity' => $level]
+            $context + [
+                'severity' => $level,
+                'resource' => $this->resource
```

Comment:
Like ensuring there is a type key? I think I would rather defer something like that to a system that validates the requests before they are delivered like we have spoken about before.